projects/manager/.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
+    "extends": "../../.eslintrc.json",
+    "ignorePatterns": ["!**/*"],
+    "overrides": [
+        {
+            "files": ["*.ts"],
+            "parserOptions": {
+                "project": ["projects/manager/tsconfig.app.json", "projects/manager/tsconfig.spec.json"],
+                "createDefaultProgram": true
+            },
+            "rules": {}
+        },
+        {
+            "files": ["*.html"],
+            "rules": {}
+        }
+    ]
+}
```

Comment:
The left contains a list of entries… e.g., dataset. But it is not highlighted at which menu point you are.

Reply:
yeah but on the right it says "Dataset details". Doesn't that suffice? 

If not, how would you want it highlighted? Another green background is too much I guess. Maybe just change the font to bold?

Reply:
98422f0f7965285b7d50db6dd8804e1f26331833